langroid/agent/special/neo4j/neo4j_chat_agent.py

```diff
@@ -87,12 +87,13 @@ def __init__(self, config: Neo4jChatAgentConfig):
         self._import_neo4j()
         self._initialize_db()
         self._init_tools_sys_message()
-        self.init_state()
+        # self.init_state() # already called inside _init_tools_sys_message()
 
     def init_state(self) -> None:
         super().init_state()
         self.current_retrieval_cypher_query: str = ""
-        self.tried_schema: bool = False
+        if not self.tried_schema:
```

Comment:
this is superfluous

Reply:
yes, I should have removed this after commenting out `self.init_state()`